components/textbook-beta/TextbookBetaContentMenuSection.vue

```diff
@@ -12,11 +14,18 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
+import QiskitMegaMenuDropdown from '@qiskit-community/qiskit-vue/src/components/MegaMenuDropdown/index.vue'
```

Comment:
curious, if any reason for drilling down into the `src/` directory rather than importing from just `@qiskit-community/qiskit-vue`? 
i.e., doing similar to what you did here:
https://github.com/qiskit-community/platypus/blob/2243f7b0045c985b5a6c603c677f4f9f3d07a61f/frontend/vue/components/common/ContentMenu.vue#L14

Reply:
I'm afraid this is an issue in the way we're exporting the modules in the **qiskit-vue** library. It works in some contexts like in Platypus, but not here for instance.

There is an issue to work on that: https://github.com/qiskit-community/qiskit-vue/issues/12.

After that gets solved we can change these imports.

Reply:
oh ok, i wasnt aware. thanks!